test/tools/ossfuzz/protoToYul.cpp

```diff
@@ -966,23 +974,43 @@ void ProtoConverter::visit(FunctionCall const& _x)
 			"Proto fuzzer: Function call with too many output params encountered."
 		);
 
+		// Return early if numOutParams > number of available variables
+		if (numOutParams > numVarsInScope())
+			return;
```

Comment:
I haven't really looked through/thought through the entirety of this, so I may be wrong ;-).

But does this just ignore the whole function call if there's not enough variables to assign to?
Might it make sense to generate fresh variable names (e.g. ``x_3``, ``x_4``, ``x_5``, if it previously were only ``x_1`` and ``x_2`` of them) and generate
```
let x_3, x_4, x_5 := functionCall(...)
```
Adding ``x_3, x_4, x_5`` to ``m_currentFuncVars``?

Or something like that?

But as I said - I may very well be overlooking something and/or this might break other stuff, then it's not worth it.

Reply:
> But does this just ignore the whole function call if there's not enough variables to assign to?

You are right.

> Might it make sense to generate fresh variable names (e.g. x_3, x_4, x_5, if it previously were only x_1 and x_2 of them) and generate

Yeah, this is one option. I didn't do it this way because I wanted the fuzzer to "learn" to generate a multi variable declaration statement before such a function call. So the early return "teaches" the fuzzer that the input specification is sub-optimal.

Having said that, explicitly adding these variables as you suggest simply to never return early is also a possibility. I can also do that. Wdyt?

Reply:
Actually, the early return may not be as much of a drawback because, each function definition specification contains a boolean flag called `force_call` that prompts the proto converter to make a call as a multi variable declaration

```
let x, y := f()
```

The early return however means it may take a litlle longer for the fuzzer to go from

```
let x
// Function call specified but not done due to early return
```

to 

```
let x, y
x, y := f()
```

in the instance where `force_call` for `f()` is `false`

Reply:
Alright given that, I agree it's fine to return :-).